sunpy/spectra/spectrum.py

```diff
@@ -36,10 +36,10 @@ def __new__(cls, data, *args, **kwargs):
         return np.asarray(data).view(cls)
 
     def __init__(self, data, freq_axis):
-        if np.shape(data)[0] != np.shape(freq_axis)[0]:
-  	        raise ValueError('Dimensions of data and Frequency axis do not match')
+        if np.shape(data)[0]!= np.shape(freq_axis)[0]:
+            raise ValueError('Dimensions of data and Frequency axis do not match')
         else:
```

Comment:
Note that the 'else' clause is not actually needed.  This is because if data and freq_axis are indeed different, the code will raise an error and the code will stop running.